### ISSUE TITLE
select-only combobox example: Safari(mac) + VoiceOver combination doesn't announce the options

### ISSUE THREAD

**Issue by NabeelKabeer** [w3c/aria-practices#2060]
The [select-only combobox example](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html) is not working properly with VoiceOver and `Safari (15.0)`.

### Steps to reproduce
* Navigating to the combobox input, VO announces shortcut to open the listbox (`VO key + space`).
* Notice that this combination doesn't work, but simply pressing `space` works.
* Try to navigate through the available options, VoiceOver doesn't announce anything

### Note
* This is working properly with `Firefox (92.0)` and `Chrome (94.0)` when used with VoiceOver in `Mac (10.15.7)`

Could be a webkit specific issue, interested to know if there is any active webkit bug tracking this issue if that is the case.

**Comment by Rasubra8** [comment]
NVDA is also not announcing the selected option for the combobox in FF. 
FF version 97.0
NVDA 2021.2
Selectonly combobox example ARIA 1.2
https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html

**Comment by JAWS-test** [comment]
The problem also occurs with screen reader JAWS. The cause is a bug in Firefox that does not pass a value to the Accessibility API. So it should be solved by Firefox. Maybe the new combobox pattern with role=combobos at `<div>` instead of `<input>` is too new for Firefox.

**Comment by ZoeBijl** [comment]
> Maybe the new combobox pattern with role=combobos at <div> instead of <input> is too new for Firefox.

I think that’s a good guess. The pattern itself is still fairly new.

**Comment by a11ydoer** [comment]
This is known webkit issue.

**Comment by smhigley** [comment]
This is also a known Gecko/Firefox issue. Bugs:
https://bugzilla.mozilla.org/show_bug.cgi?id=1631693
https://bugzilla.mozilla.org/show_bug.cgi?id=1702392
https://bugzilla.mozilla.org/show_bug.cgi?id=1732284 (this bug is specific to ATK, but related)
